templates/types/streaming/fastapi/app/api/routers/vercel_response.py

```diff
@@ -1,15 +1,19 @@
 import json
```

Comment:
how does this change influence our multi agent streaming?

Reply:
Multi-agent has their own chat.py and vercel_response.py code.

Reply:
@leehuwuj is the event streaming also blocked in multi-agent? does it need to be updated?

Reply:
no, the multi-agent is working correctly - thanks to it so i found the blocking issue.